src/web/components/Card/Card.tsx

```diff
@@ -265,8 +272,9 @@ export const Card = ({
 								)}
 								<CardFooter
```

Comment:
I feel like it might be better to have `!badge` at the `CardFooter` component instead of having it for each prop? You could then add use `LabBadge` component here instead of `CardFooter`?

Reply:
I was told to do it this way 🤷‍♂️